source/ed/connectors/fusio_parser.cpp

```diff
@@ -346,14 +346,12 @@ std::vector<ed::types::VehicleJourney*> TripsFusioHandler::get_split_vj(Data& da
     // get shape if possible
     const std::string &shape_id = has_col(geometry_id_c, row) ? row.at(geometry_id_c) : "";
 
-    const auto vp_end_it = gtfs_data.tz.vp_by_name.upper_bound(row[service_c]);
-
-    auto second_elt = vp_it;
-    bool has_been_split = (++second_elt != vp_end_it); //check if the trip has been split over dst
+    auto second_elt = vp_range.second;
+    bool has_been_split = (++second_elt != vp_range.second); //check if the trip has been split over dst
```

Comment:
Same here, abstract that, you want more_than_one_elt:
```c++
template<typename T> bool more_than_one_elt(const std::pair<T, T>& r) {
    return ! empty(r) && boost::next(r.first) != r.second;
}
```
See http://www.boost.org/doc/libs/1_57_0/libs/utility/utility.htm#functions_next_prior for `boost::next`